cmake/SetupPypp.cmake

```diff
@@ -0,0 +1,27 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# To run Python tests we need the development component (libs and include dirs).
+# It provides the `Python::Python` imported target.
+find_package(Python REQUIRED COMPONENTS Development)
+
+# We can't rely on CMake 3.14 quite yet so we backport the `Python::NumPy`
+# imported target that FindPython in CMake 3.14+ could also provide.
+find_package(NumPy 1.10 REQUIRED)
```

Comment:
Do we/can we check that SciPy was correctly found with finding NumPy? I believe nowadays it is, but I don't know when that changed.

Reply:
Probably a good idea to check that, added a fixup